nixos/modules/security/pwquality/pwquality-conf-format.nix

```
@@ -0,0 +1,274 @@
+{ pkgs, lib, ... }:
+
+let
+  inherit (lib) types;
+in
+
+{
+  type = types.submodule {
+    freeformType =
+      with types;
+      (attrsOf (
+        nullOr (oneOf [
+          str
+          int
+          bool
```

(Continuing my suggestions for false-like values below)

```suggestion
          # false is not explicitly represented in the config.
          # While we are confident about the translation of the existing explicit boolean options below,
          # we aren't as completely confident what the behavior for `false` should be for unknown,
          # presumably new keys.
          # If you're having trouble with this, add the new option below.
          (enum [ true ])
```